lib/commands/exam.js

```diff
@@ -332,6 +334,44 @@ module.exports = TestCommand.extend({
     }
   },
 
+  /**
+   * Adds an object which maps a list of module from browser id to config
+   * 
+   * @param {object} commandOptions
+   * @param {object} config
+   */
+  _addBrowserModuleMappingToConfig: function(commandOptions) {
+    const filePath = commandOptions.replayExecution;
+    const replayBrowser = commandOptions.replayBrowser;
+
+    let testMappingObj = {}
+
+    try {
+      // Read the replay execution json file.
+      const testMappingRawData = fs.readFileSync(filePath);
+      testMappingObj = JSON.parse(testMappingRawData);
+    } catch (err) { 
+      throw new Error(err);
```

Comment:
I'm not sure I'd directly throw this error. I might wrap it in an error that says something like "error reading replay execution JSON file", then the error string from this error. It's clearer to the user that way.